ReactWindows/ReactNative.Shared/UIManager/BaseViewManager.cs

```diff
@@ -58,6 +67,7 @@ public void SetOpacity(TFrameworkElement view, double opacity)
         [ReactProp("overflow")]
         public void SetOverflow(TFrameworkElement view, string overflow)
         {
+#if WINDOWS_UWP
             if (overflow == "hidden")
```

Comment:
I think you want to disable this native prop altogether if you're not going to do anything with it.